workflows/prognostic_c48_run/tests/_regtest_outputs/test_prepare_config.test_prepare_ml_config_regression[emulator].out

```diff
@@ -427,12 +427,7 @@ namelist:
     fvmxl: 99999
     ldebug: false
 nudging: null
-online_emulator:
-  emulator: some/path
-  ignore_humidity_below: null
-  mask_kind: default
-  online: false
-  train: true
+online_emulator: null
```

Comment:
Would be better to delete the `emulator` prepare config test instead of updating the test.

Reply:
This test also tests the zhao_carr_emulation integration.

Reply:
Got it.